apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -49,6 +50,21 @@ export function HeaderNav({}: Props) {
     }, [currentUser, currentOrganization]);
   }
 
+  if (LOGROCKET_ID) {
+    useEffect(() => {
+      if (currentUser && currentOrganization) {
+        let email = currentUser?.email;
+        email ??= ' ';
```

Comment:
What is this `??=` never seen such syntax in js. 

Also why are we doing an empty space string here instead of `''`? Would prefer to leave it as null/undefined or even empty string so it will be considered a falsy value.

```suggestion
        const email = currentUser?.email ?? '';
```

Reply:
It is the nullish coalescing operator. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
It is useful to check for existence without being triggered by 0 or false.

Reply:
The reason I did this is that logrocket traits does not support undefined value which '' is also considered undefined.